### DISCUSSION TITLE
i do have win10 i download  xlights

### DISCUSSION THREAD

**Discussion by icemanrj** [xLightsSequencer/xLights#4746] [Q&A] [answered]
I installed xlights correctly but when I go to open it it doesn't open, my windows is win10 64bit can someone help me, the other files like scan open but the application doesn't

**Comment by cybercop23** [comment]
There are a few open/close issues here related to this. Try install ing this and reboot and see if it starts.
https://aka.ms/vs/17/release/vc_redist.x64.exe

Info in teh Releases page... https://xlights.org/releases/

**Reply by icemanrj** [reply]
sure i will try but i did everyting , install , remove install again

**Reply by icemanrj** [reply]
i install that also no work

**Reply by cybercop23** [reply, marked as answer]
xlightszoom.com. make sure you have the zoom app installed and we can take a look.

**Reply by icemanrj** [reply]
> xlightszoom.com. make sure you have the zoom app installed and we can take a look.

i click on the link im on the zoom ok let me know how works

**Reply by cybercop23** [reply]
You don't have a mic.. so use the  dial in options under the Audio button on the left lower side.

**Reply by icemanrj** [reply]
> You don't have a mic.. so use the dial in options under the Audio button on the left lower side.

yes i do have a mic and works there i saw 7 people on the zoom
